### PR TITLE
Derive Default for all measurement structs

### PR DESCRIPTION
**Pull request by eta077**

### CODE REVIEW

**Review by eldruin — COMMENTED** [review]
Looks fine to me, thanks!
Could you add an entry to the changelog?

**Comment by eta077** [comment]
Absolutely, sorry I didn't catch that - thanks for the review!

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,10 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+## Changed
+
+- Derive Default for all measurement structs
```
Comment (eldruin) [inline CHANGELOG.md:11]:
```suggestion
## Added

- Derive `Default` for all measurement structs
```

**Review by eldruin — COMMENTED** [review]
Looks good to me, thanks!

**Pull request merged**